### PR TITLE
[native] Cosmetic changes in https path

### PR DESCRIPTION
**Pull request by amitkdutta**
Addressing comments from https://github.com/prestodb/presto/pull/19371

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

File: presto-native-execution/presto_cpp/main/http/HttpServer.cpp [hunk 1]
```diff
@@ -89,7 +89,7 @@ HttpsConfig::HttpsConfig(
   std::replace(supportedCiphers_.begin(), supportedCiphers_.end(), ',', ':');
 }
 
-proxygen::HTTPServer::IPConfig HttpsConfig::getHttpsConfig() const {
+proxygen::HTTPServer::IPConfig HttpsConfig::getHttpsIpConfig() const {
```
Comment (xiaoxmeng) [inline presto-native-execution/presto_cpp/main/http/HttpServer.cpp:92]:
```
s/getHttpsIpConfig/httpsIpConfig/ we don't need get prefix for getter method in a class. 
```

**Review by xiaoxmeng — APPROVED** [review]
@amitkdutta thanks for taking care of these!

File: presto-native-execution/presto_cpp/main/common/Configs.h [hunk 2]
```diff
@@ -172,12 +172,28 @@ class SystemConfig : public ConfigBase {
 
   int httpServerHttpsPort() const;
 
+  // This config controls what cipher suites are supported by Native workers for
```
Comment (mbasmanova) [inline presto-native-execution/presto_cpp/main/common/Configs.h:175]: > // This config controls what cipher suites a

Consider to shorten to "A list of supported ciphers."

The return type is a std::string. Does it contain the list of ciphers separated by comma or something? Would be nice to clarify.

Comment (amitkdutta) [reply presto-native-execution/presto_cpp/main/common/Configs.h:175]: Yes. It's a comma separated list like `AES128-SHA,AES128-SHA256,AES256-GCM-SHA384`

**Pull request merged**